src/Tizen.Multimedia.MediaCodec/MediaCodec/MediaCodec.cs

```diff
@@ -617,11 +618,7 @@ private void RegisterBufferStatusChanged()
         {
             _bufferStatusCb = (statusCode, _) =>
             {
-                Debug.Assert(Enum.IsDefined(typeof(MediaCodecStatus), statusCode),
-                    $"{ statusCode } is not defined in MediaCodecStatus!");
```

Comment:
even if the type is changed to enum, the range of it will not be checked. are there any reasons to delete it?

Reply:
It's p/invoke function so, it's hard to set wrong value be native framework.
But, even if so, it's better to check it. I'll rollback it.